modules/geo-layers/src/h3-layers/h3-hexagon-layer.js

```diff
@@ -1,7 +1,21 @@
-import {h3ToGeoBoundary, h3GetResolution, h3ToGeo, geoToH3, h3IsPentagon} from 'h3-js';
+import {
+  h3ToGeoBoundary,
+  h3GetResolution,
+  h3ToGeo,
+  geoToH3,
+  h3IsPentagon,
+  h3Distance,
+  edgeLength,
+  UNITS
+} from 'h3-js';
 import {CompositeLayer, createIterable} from '@deck.gl/core';
 import {ColumnLayer, PolygonLayer} from '@deck.gl/layers';
 
+// There is a cost to updating the instanced geometries when using highPrecision: false
+// This constant defines the distance between two hexagons that leads to "significant
+// distortion." Smaller value makes the column layer more sensitive to viewport change.
+const UPDATE_THRESHOLD = 10000; // 10km
```

Comment:
Nit: When using unit constants, it's usually good to have the unit in the name. In this case, I'd use `km` instead of `m`, and go with
```
const UPDATE_THRESHOLD_KM = 10;
```
Then you don't need the comment and the unit info is available at the point of use.